app/main.py

```diff
@@ -1 +1,21 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+
+    command_name, src_file, dest_file = command.split(" ")
+
+    if command_name == "mv" and os.path.exists(src_file):
+
+        dest_list = dest_file.split("/")
```

Comment:
It's better to use os.path.split. For example, on Windows we have `\` instead of `/`